README.md

```diff
@@ -11,45 +11,119 @@ curl https://raw.githubusercontent.com/code-423n4/Bot-Race-JSON-Schema/v0.1.0/sc
 npx ajv-cli --spec=draft2020 -s ./schema.json -d report.json
 ```
 
-### Upcoming Version 0.2.0
+### Upcoming Version 0.3.0
 
-A new version of the schema will replace v0.1.0 soon (dates to be announced) and can be found [here](https://github.com/code-423n4/Bot-Race-JSON-Schema/blob/v0.2.0/schema.json).
-This version restricts the values of `severity`, making it an enum instead of an open string.
+A new version of the schema will replace v0.1.0 on [Place Holder Date - TBD] and can be found [here](https://github.com/code-423n4/Bot-Race-JSON-Schema/blob/v0.3.0/schema.json).
+**Changes**:
+
+- restricts the values of `severity`, making it an enum instead of an open string.
+- `content` and `description` are optional
+- `content` must be in code block format, we are expecting \`\`\` at the begining and end of this string.
+- `loc` must contain a link to the code or file.
 
 If you want to test your report against the upcoming schema you can fetch the latest tag and validate as before:
 
 ```
-curl https://raw.githubusercontent.com/code-423n4/Bot-Race-JSON-Schema/v0.2.0/schema.json -o schema.json
+curl https://raw.githubusercontent.com/code-423n4/Bot-Race-JSON-Schema/v0.3.0/schema.json -o schema.json
 npx ajv-cli --spec=draft2020 -s ./schema.json -d report.json
```

Comment:
Just a reminder to myself we'll want to update the examples when we have a lib released.